src/status_im/react_native/resources.cljs

```diff
@@ -2,52 +2,53 @@
+   :lock                   (js/require "../resources/images/ui/lock.png")
+   :empty-wallet           (js/require "../resources/images/ui/empty-wallet.png")
+   :tribute-to-talk        (js/require "../resources/images/ui/tribute-to-talk.png")
+   :keycard-card           (js/require "../resources/images/ui/hardwallet-card.png")
+   :keycard-lock           (js/require "../resources/images/ui/keycard-lock.png")
+   :keycard                (js/require "../resources/images/ui/keycard.png")
+   :keycard-logo           (js/require "../resources/images/ui/keycard-logo.png")
+   :keycard-logo-blue      (js/require "../resources/images/ui/keycard-logo-blue.png")
+   :keycard-logo-gray      (js/require "../resources/images/ui/keycard-logo-gray.png")
+   :keycard-key            (js/require "../resources/images/ui/keycard-key.png")
+   :keycard-empty          (js/require "../resources/images/ui/keycard-empty.png")
+   :keycard-phone          (js/require "../resources/images/ui/keycard-phone.png")
+   :keycard-connection     (js/require "../resources/images/ui/keycard-connection.png")
+   :keycard-wrong          (js/require "../resources/images/ui/keycard-wrong.png")
+   :not-keycard            (js/require "../resources/images/ui/not-keycard.png")
+   :status-logo            (js/require "../resources/images/ui/status-logo.png")
+   :warning-sign           (js/require "../resources/images/ui/warning-sign.png")
+   :phone-nfc-on           (js/require "../resources/images/ui/phone-nfc-on.png")
+   :phone-nfc-off          (js/require "../resources/images/ui/phone-nfc-off.png")
+   :dapp-store             (js/require "../resources/images/ui/dapp-store.png")
+   :ens-header             (js/require "../resources/images/ui/ens-header.png")
+   :ens-header-dark        (js/require "../resources/images/ui/ens-header-dark.png")
+   :unfurl                 (js/require "../resources/images/ui/unfurl.png")
+   :unfurl-dark            (js/require "../resources/images/ui/unfurl-dark.png")
+   :new-chat-header        (js/require "../resources/images/ui/new-chat-header.png")
+   :onboarding-phone       (js/require "../resources/images/ui/onboarding-phone.png")
+   :theme-dark             (js/require "../resources/images/ui/theme-dark.png")
+   :theme-light            (js/require "../resources/images/ui/theme-light.png")
+   :theme-system           (js/require "../resources/images/ui/theme-system.png")
+   :notifications          (js/require "../resources/images/ui/notifications.png")
+   :collectible            (js/require "../resources/images/ui/collectible.png")
+   :collectible-dark       (js/require "../resources/images/ui/collectible-dark.png")
+   :hand-wave              (js/require "../resources/images/ui/hand-wave.png")
+   :graph                  (js/require "../resources/images/ui/graph.png")
+   :switcher               (js/require "../resources/images/ui/switcher.png")
+   :default-account-avatar (js/require "../resources/images/ui/default-account-avatar20.png")})
```

Comment:
why is it in images but not in the icons ?

Reply:
Hmmm no preference, It's actually a temporary fix till we have a seperate emoji component to import all emojis